### PR TITLE
ON-17 Added a database to Ontology API

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: ontology_api/model/model_router.py [hunk 1]
```diff
@@ -20,28 +20,40 @@ class ModelRouter:
...
+        if create_response.errors is not None:
+            response.status_code = 422
+
+        # add links from hateoas
+        create_response.links = get_links(router)
+        return create_response
+
     @router.get("/model/{id}", tags=["model"])
-    async def get_owl(self, id: int, response: Response, background_tasks: BackgroundTasks):
+    async def get_owl(self, id: int, response: Response):
```
Comment (mrwrob) [inline ontology_api/model/model_router.py:56]: When you removed `BackgroundTasks`, please remove relevant import as well.

File: ontology_api/tests/test_model_router.py [hunk 2]
```diff
@@ -11,49 +11,40 @@ class ModelRouterTestCase(unittest.TestCase):
     def test_get_owl_without_error(self):
         model_router = ModelRouter()
         response = Response()
-        background_tasks = BackgroundTasks()
-        model_router.model_service.models[1] = get_ontology("http://www.semanticweb.org/GRISERA/contextualOntology")
-        result = asyncio.run(model_router.get_owl(1, response, background_tasks))
-        #os.remove("tmp_owl/contextualOntology1.owl")
+        model = get_ontology("http://www.semanticweb.org/GRISERA/contextualOntology")
```
Comment (mrwrob) [inline ontology_api/tests/test_model_router.py:14]: Check unused imports.

**Review by mrwrob — CHANGES_REQUESTED** [review]
Seems fine, please optimize imports.

**Review by mrwrob — APPROVED** [review]
Seems fine :+1:

**Review by KKucht — APPROVED** [review]
I believe in this man

**Pull request merged**